pom.xml

```diff
@@ -193,63 +193,63 @@
             </plugin>
 
             <!-- Enforce check styles validation -->
```

Comment:
Commenting out sections that weren't working, but I haven't spent time looking into them yet!